src/Utility.ts

```diff
@@ -20,6 +20,7 @@ export class Utility {
    */
   public async setAzureCliDynamicInstall() {
     await this.execute(`az config set extension.use_dynamic_install=yes_without_prompt`);
```

Comment:
nit: we shouldn't need this command anymore if we're explicitly installing the `containerapp` extension below